scripts/check-prebuild.js

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+
+/* eslint-disable import/no-commonjs */
```

Comment:
lets add a overrides in the eslint config to disable commonjs

Reply:
Fixed in [a54cc25](https://github.com/react-navigation/react-navigation/pull/11020/commits/a54cc253ee77c91f19d54fbb302ef666ddaf0886)